README.md

```diff
@@ -57,7 +57,7 @@ This will run the fabric command to deploy to production.
 The command to update the data in Pulse and publish it to production is simple:
 
 ```
-./data/update
+python -m data.update
```

Comment:
Note the weird syntax for executing the data script. This is done so that the `data/update.py` file can reach "up" and back down into its sibling `app/` directory, which is not otherwise possible in Python 3.